components/layer-legend.vue

```diff
@@ -60,7 +60,7 @@
     props: {
       unit: {
         type: String,
-        required: true,
+        default: () => '',
```

Comment:
would `default: ''` also work?
Why is default not '-'?

Reply:
Because this is showing next to the min max values, this might be a bit much, because the unit is also shown in a seperate text field.